src/content/en/2019/markup.md

```diff
@@ -0,0 +1,301 @@
+
+Among the probably not challenging, but popular ones are some interesting entries:
+
+[`<ymaps>` (from yahoo maps) appears on ~12.5k mobile sites](https://rainy-periwinkle.glitch.me/permalink/2ba66fb067dce29ecca276201c37e01aa7fe7c191e6be9f36dd59224f9a36e16.html) ([~8.3k desktop](https://rainy-periwinkle.glitch.me/permalink/7f365899dc8a5341ed5c234162ee4eb187e99a23fc28cdea31af2322029d8b48.html))
+
+[`<cufon>` and `<cufontext>` from a font replacement library from 2008, appear on ~10.5k of mobile pages ](https://rainy-periwinkle.glitch.me/permalink/5cfe2db53aadf5049e32cf7db0f7f6d8d2f1d4926d06467d9bdcd0842d943a17.html)(~[8.7k desktop](https://rainy-periwinkle.glitch.me/permalink/c9371b2f13e7e6ff74553f7918c18807cd9222024d970699e493b2935608a5f2.html)) 
+
+There is also [the `<jdiv>` element appears to be injected by Jivo chat, a popular chat solution which appears on ~40.3k of mobile sites](https://rainy-periwinkle.glitch.me/permalink/976b0cf78c73d125644d347be9e93e51d3a9112e31a283259c35942bda06e989.html) ([~37.6k of desktop pages - that's roughly ~0.86%)](https://rainy-periwinkle.glitch.me/permalink/98fb3bf4f44c33edabc05439b10a374a121dbbfc5f83af65e00e859039b13acd.html)!
+
+ 
+
+Placing these into our same chart as above for perspective looks something like this (again, it varies slightly based on the dataset)
+
+
+(( TODO: there is a corresponding image in the google doc https://docs.google.com/document/d/16TY_pV-FyW35DzuvdlaOiENz6o6PWYpl_RviU-HW7Qc/edit ))
+
+The interesting thing about these is that they also introduce a few other ways that our tool can come in very handy:  If we're interested in exploring the space of the data, a very specific tag name is just one possible measure.  It's definitely the strongest indicator if we can find good slang developing.  However, what if that's not all we're interested in? 
+
+### Common use cases and solutions
+
+What if, for example, we were interested in people solving common use cases?  This could be because we're looking for solutions to use cases that we currently have ourselves, or for researching more broadly what common use cases people are solving with an eye toward incubating some standardization effort.  Let's take a common example: Tabs.  Over the years there have been a lot of requests for things like tabs.  We can use a fuzzy search here and find that there are [many variants of tabs](https://rainy-periwinkle.glitch.me/permalink/c6d39f24d61d811b55fc032806cade9f0be437dcb2f5735a4291adb04aa7a0ea.html).  It's a little harder to count use here since we can't as easily distinguish if two elements appear on the same page, so the count provided there conservatively simply takes the one with the largest count - in most cases the real number of domains is probably significantly larger.
```

Comment:
```suggestion
What if, for example, we were interested in people solving common use cases?  This could be because we're looking for solutions to use cases that we currently have ourselves, or for researching more broadly what common use cases people are solving with an eye toward incubating some standardization effort.  Let's take a common example: Tabs.  Over the years there have been a lot of requests for things like tabs.  We can use a fuzzy search here and find that there are [many variants of tabs](https://rainy-periwinkle.glitch.me/permalink/c6d39f24d61d811b55fc032806cade9f0be437dcb2f5735a4291adb04aa7a0ea.html).  It's a little harder to count use here since we can't as easily distinguish if two elements appear on the same page, so the count provided there conservatively simply takes the one with the largest count -- in most cases the real number of domains is probably significantly larger.
```